app/Http/Controllers/Api/Entries/DownloadSubsetController.php

```
@@ -154,14 +154,10 @@ private function createSubsetArchive(Builder $query, string $filename): string
 
-        //get handle of empty file just created
-        $CSVfilepath = Storage::disk('temp')
-                ->path('')
-            . 'subset/' . $projectRef . '/' . $csvFilename;
-
-        $zipFilepath = Storage::disk('temp')
-                ->path('')
-            . 'subset/' . $projectRef . '/' . $zipFilename;
+        //get handle of empty files just created
+        $diskRoot = config('filesystems.disks.temp.root').'/';
+        $CSVfilepath = $diskRoot . 'subset/' . $projectRef . '/' . $csvFilename;
```

PHPMD (2.15.0)

The variable $CSVfilepath is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)